test/pugtester.js

```diff
@@ -7,15 +7,15 @@ const compiledFunction = pug.compileFile('views/snippet.pug');
 console.log(compiledFunction({
```

Comment:
Should the name of this file be pugTester.js? To keep it consistent w/ e.g. gistProvider.